src/audiomixerboard.h

```diff
@@ -182,65 +196,68 @@ class CAudioMixerBoardSlots : public CAudioMixerBoardSlots<slotId - 1>
+    void    HideAll();
+    void    ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInfo );
+    void    SetServerName ( const QString& strNewServerName );
+    QString GetServerName() { return strServerName; }
+    void    SetGUIDesign ( const EGUIDesign eNewDesign );
+    void    SetDisplayPans ( const bool eNDP );
+    void    SetPanIsSupported();
+    void    SetRemoteFaderIsMute ( const int iChannelIdx, const bool bIsMute );
+    void    SetMyChannelID ( const int iChannelIdx )
+    {
+        iMyChannelID = iChannelIdx;
+    }
 
-    void            SetFaderLevel ( const int iChannelIdx,
-                                    const int iValue );
+    void SetFaderLevel ( const int iChannelIdx, const int iValue );
 
-    void            SetPanValue ( const int iChannelIdx,
-                                  const int iValue );
+    void SetPanValue ( const int iChannelIdx, const int iValue );
 
-    void            SetFaderIsSolo ( const int iChannelIdx,
-                                     const bool bIsSolo );
+    void SetFaderIsSolo ( const int iChannelIdx, const bool bIsSolo );
 
-    void            SetFaderIsMute ( const int iChannelIdx,
-                                     const bool bIsMute );
+    void SetFaderIsMute ( const int iChannelIdx, const bool bIsMute );
 
-    void            SetNumMixerPanelRows ( const int iNNumMixerPanelRows );
-    int             GetNumMixerPanelRows() { return iNumMixerPanelRows; }
+    void SetNumMixerPanelRows ( const int iNNumMixerPanelRows );
+    int  GetNumMixerPanelRows() { return iNumMixerPanelRows; }
 
-    void            SetFaderSorting ( const EChSortType eNChSortType );
-    EChSortType     GetFaderSorting() { return eChSortType; }
+    void        SetFaderSorting ( const EChSortType eNChSortType );
+    EChSortType GetFaderSorting() { return eChSortType; }
 
-    void            SetChannelLevels ( const CVector<uint16_t>& vecChannelLevel );
+    void SetChannelLevels ( const CVector<uint16_t>& vecChannelLevel );
 
-    void            SetRecorderState ( const ERecorderState newRecorderState );
-    ERecorderState  GetRecorderState() { return eRecorderState; };
+    void           SetRecorderState ( const ERecorderState newRecorderState );
+    ERecorderState GetRecorderState() { return eRecorderState; };
 
-    void            SetAllFaderLevelsToNewClientLevel();
-    void            StoreAllFaderSettings();
-    void            LoadAllFaderSettings();
-    void            AutoAdjustAllFaderLevels();
+    void SetAllFaderLevelsToNewClientLevel();
+    void StoreAllFaderSettings();
+    void LoadAllFaderSettings();
+    void AutoAdjustAllFaderLevels();
 
-protected:
+  protected:
     class CMixerBoardScrollArea : public QScrollArea
     {
-    public:
-        CMixerBoardScrollArea ( QWidget* parent = nullptr ) : QScrollArea ( parent ) {}
+      public:
+        CMixerBoardScrollArea ( QWidget* parent = nullptr ) :
```

Comment:
Again, if not too wide, single line preferred.  However, that parent class reference may take it over.  Let's say it does.  In that case, the `{}` would be on the same line as the parent reference.  I think...

Reply:
also fixed with `AllowShortBlocksOnASingleLine: Empty`